resources/todomvc/big-dom-generator/src/app.css

```diff
@@ -86,6 +86,14 @@ Importing the CSS from the Spectrum CSS library.
     min-width: 30px;
 }
 
+.display-none {
+    display: none;
+}
+
+.spectrum-TreeView-item.is-open > .spectrum-TreeView {
+    visibility: unset;
+}
```

Comment:
The impact of this CSS class is not clear to me. Which node does this apply to? Can you please add some more comments around that?

Reply:
Added this comment:
```
/*
The spectrum rule making the open tree view items visible is
.spectrum-TreeView-item.is-open > .spectrum-TreeView {
    visibility: visible;
}
This can cause descendants of closed tree view items to be visible.
We unset here to let it inherit the visibility of its parent.
*/
```

This is an example of the pathological behavior:
<img width="104" alt="image" src="https://github.com/WebKit/Speedometer/assets/94007365/bb7a7555-e184-41cb-9c3f-76ac7d3fb60d">

Reply:
I see, but why wasn't this happening before?

Reply:
> I see, but why wasn't this happening before?

Because before, our code was assuming that every tree item under a closed item is also closed, I changed it to only close the items at the threshold level. It makes more sense to me that the visibility of the children depends on the visibility of the parent instead of relying on its own `is-open` value.

Reply:
I see, can you write it as a comment too? I assume it's not just your code but spectrum in general assuming this.

Reply:
Done.